[DOC] Update preview second and add edit section (#12864)
Scille/parsec-cloud: docs/userguide/manage-files.rst

Parsec preview is limited to files with size inferior to 15MB. The following file formats are supported:

* PDF
* DOCX (Microsoft Word), ODT (OpenOffice)
* XLSX (Microsoft Excel), ODS (OpenOffice)
* Audio
* Images
* PPTX (Microsoft Powerpoint), ODP (OpenOffice) support is coming soon.
* Plain Text and Code file formats: TXT, MD, CSV, RST, JSON...

On the desktop app, Parsec preview can be disabled in the application settings.

.. image:: screens/files/disable_viewers.png
    :align: center
    :alt: Disable viewers screenshot
    :width: 800

> Using Parsec edition

Reviewer: This is a faux-amis: _édition de fichiers_ (FR) translates to "file editing" (not "file edition"). 
See https://english.stackexchange.com/questions/44238/can-edition-be-used-to-mean-the-act-of-editing-changing-something

So it should be
```suggestion
Using Parsec editor
```
or maybe rework title to
```suggestion
File editing in Parsec 
```

I prefer the former, but it might be misleading since it is essentially a CryptPad (not Parsec) editor.

^^^^^^^^^^^^^^^^^^^^

> Some files can be edited directly in Parsec without using an external application. The editor is available to **Owners**, **Managers** and **Contributors** of a workspace, by clicking on ``Edit`` in the file menu, clicking the file name or double-clicking the file in Parsec.

Reviewer: Note: I would add some spacing to make it easier to read.

```suggestion
Some files can be edited directly in Parsec without using an external application. 

The editor is available to **Owners**, **Managers** and **Contributors** of a workspace, by clicking on ``Edit`` in the file menu, clicking the file name or double-clicking the file in Parsec.
```

.. image:: screens/files/context_edit.png
    :align: center
    :alt: Context menu edit

> If the file type is supported (see below), Parsec will open it in a embedded editor.

Reviewer: ```suggestion
If the file type is supported (see below), Parsec will open it in the editor.
```

.. image:: screens/files/docx_editor.png
    :align: center
    :alt: DOCX editor screenshot

> Parsec edition is limited to files with size inferior to 15MB. The following file formats are supported:

Reviewer: File _editing_ + spacing

```suggestion
File editing in Parsec is limited to files with size inferior to 15MB. 

The following file formats are supported:
```

* Documents: DOCX (Microsoft Word), ODT (OpenOffice)
* Spreadsheet: XLSX (Microsoft Excel), XLS, ODS (OpenOffice)

> * Presentation: PPTX (Microsoft Powerpoint), ODP (OpenOffice) support is coming soon.

Reviewer: I would use the same structure than Preview: either add above "Documents:", "Spreadsheet:", "Presentation:", OR remove it here.

* Plain Text and Code file formats: TXT, MD, CSV, RST, JSON...

.. note::

>     Collaborative edition is currently in development. Until the official release of this feature, only the latest change will be saved and overwrite the file if two users edit a file simultaneously, regardless of conflicts.

Reviewer: It's not collaborative (yet). Also I wouldn't mention "official release".

```suggestion
    File editing is currently in active development. For the moment, only the latest change will be saved and overwrite the file if two users edit a file simultaneously, regardless of conflicts.
```

Reviewer: Are you sure about overwrite/conflicts? Doesn't it work like editing with an external app? i.e. Alice saves first, changes are applied, Bob saves later, a conflict is detected, a copy of the file is created with suffix `Parsec - content conflict`

https://github.com/Scille/parsec-cloud/blob/3ee2280da97cfa92d8cc04944b94a2bff900f793/libparsec/crates/client/src/workspace/merge.rs#L10-L11

> On the desktop app, Parsec edition can be disabled in the application settings using the same option shown for preview.

Reviewer: ```suggestion
On the desktop app, file editing in Parsec can be disabled from the application settings using the same option shown for preview.
```

Maybe we can include the same image here for completeness?

```suggestion
On the desktop app, file editing in Parsec can be disabled from the application settings using the same option shown for preview.

.. image:: screens/files/disable_viewers.png
    :align: center
    :alt: Disable file editing in Parsec screenshot
    :width: 800
```
